README.md

```diff
@@ -1,2 +1,127 @@
+
+⚫ NFT ownership verification
+
+## Documentation
+
+### Attendify.js
+
+⚫ Current version of documentation for Attendify.js is in the `docs` folder and was generated by jsdoc. Latest version of docs is hosted [here](https://justanotherdevv.github.io/AttendifyXRPL/Attendify.html).
+
+### REST API endpoints
+
+This API is built using the ExpressJS framework and the XRPL library, it connect to the local Attendify.js class to perform most of the actions.
+
+⚫ `GET /api/mint` - Creates new claimable event and premints NFTs for it. You must have at least X XRP in order to use this to handle the reserve requirements for the NFTs and the minting process which uses [Tickets](https://xrpl.org/tickets.html), where X is amount of tokens that should be minted multiplied by 2 + additional XRP for feees.
+
+**@param {string} walletAddress** - Wallet address from user requesting claim
+
+**@param {integer} tokenCount** - Amount of NFTs that should be created for the event
+
+**@param {string} url** - Url to Image for event | Preferably this should be stored on IPFS
+
+**@param {string} title** - Title for event
+
+**@param {string} desc** - Description of event
+
+**@param {string} loc** - Location of event
+
+Example request `http://localhost:4000/api/mint?walletAddress=rpHJL8bRW81shYm3LwwsXovABps6SM51RS&tokenCount=5&url=ipfs://QmQDDD1cNgnyhPC4pBLZKhVeu12oyfCAJoWr1Qc1QgbkPN&title=test_title&desc=test_description&loc=Warsaw`
+
+⚫ `GET /api/claim` - Checks if user is eligible for NFT claim or creates new offer that allows for claiming NFT for specific event. Endpoint returns response with sellOfferID that has to be accepted by user afterwards.
+
+**@param {string} walletAddress** - The wallet address of the user trying to claim
+
+**@param {integer} type** - The type of claim (1 for checking claim status and metadata, 2 for claiming)
+
+**@param {string} minter** - The minter address of the event
+
+**@param {string} eventId** - The ID of the event
+
+**Possible responses**
+
+    -`404` - Event with specified ID does not exist
+
+    -`claimed` - The NFT for specified event was already claimed by this user
+
+    -`empty` - All NFTs for this particular event were already claimed
+
+    -`success` - If `onlyCheckStatus` parameter was true it indicated that current parameters for selected event were sent
+
+    -`transferred` - Indicates that new sell offer for NFT related to selected event was created successfully and details were sent to user
+
+Example request `http://localhost:4000/api/claim?walletAddress=rpHJL8bRW81shYm3LwwsXovABps6SM51RS&type=2&minter=rMBU1wwAJZZz36YBNxug7YYHKeC5k3vqfG&eventId=0`
+
+⚫ `GET /api/startVerification` - Starts ownership verification process by generating unique ID for user that has to later be included in a Memo of signed tx.
+
+**@param {string} walletAddress** - Wallet address from user requesting verification
+
+Example request `http://localhost:4000/api/startVerification?walletAddress=raY33uxEbZFg7YS1ofFRioeENLsVdCgpC5`
```

Comment:
For `startVerification` can you also provide an example transaction that can be signed to show how to add it to the memo? (There are a lot of different ways of putting it into the memo - should it be hex encoded, normal string, etc.)